editor/property_selector.cpp

```diff
@@ -210,6 +210,9 @@ void PropertySelector::_update_search() {
 
 				methods.push_back(MethodInfo("*Script Methods"));
 				Object::cast_to<Script>(obj)->get_script_method_list(&methods);
+				// TODO: Filter out things unsuitable for explicit calls, like constructors.
```

Comment:
This is language dependent so I would move this comment to `CSharpScript::get_script_method_list`.

Reply:
Done.